content/en/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping.md

```diff
@@ -220,16 +448,24 @@ referencing the generated key and obtained certificate is written to the path
 specified by `--kubeconfig`. The certificate and key file will be placed in the
 directory specified by `--cert-dir`.
 
-{{< note >}}
-The following flags are required to enable this bootstrapping when starting the kubelet:
+### Client and Serving Certificates
+All of the above relate to kubelet _client_ certificates, specifically, the certificates a kubelet
+uses to authenticate to kube-apiserver.
 
-```
---bootstrap-kubeconfig="/path/to/bootstrap/kubeconfig"
-```
-{{< /note >}}
+A kubelet also can use _serving_ certificates. The kubelet itself exposes an https endpoint for certain features.
+To secure these, the kubelet can do one of:
 
-Additionally, in 1.7 the kubelet implements __beta__ features for enabling
-rotation of both its client and/or serving certs.  These can be enabled through
+* use provided key and certificate, via the `--tls-private-key-file` and `--tls-cert-file` flags
+* create self-signed key and certificate, if a key and certificate are not provided
```

Comment:
this list is confusing in combination with the section below... I would include a third option: `* request serving certificates from the cluster signer, via the CSR API`